lib/active_model_serializers/model.rb

```diff
@@ -30,6 +30,8 @@ def updated_at
     end
 
     def read_attribute_for_serialization(key)
+      return send(key) if respond_to?(key)
```

Comment:
Nope, I was waiting to get approval of the RFC. Because what would that module be? Would it contain the constructor logic (in which case we are in the business of overriding the superclass's constructor which is gross)? And if not, what *would* it include?

As far as I can tell it would just be `alias read_attribute_for_serialization send`, and in that case I'm not sure what the point of it is.

So, TBH AMS::Model is nonsensical to me, which means I don't think I can touch the code or enhance the documentation in a meaningful way. I created the RFC to try and improve that situation, but doesn't look like it is getting any traction.